controllers/cards.js

```diff
@@ -0,0 +1,44 @@
+const Card = require('../models/card');
+
+// GET /cards — возвращает все карточки
+const getAllCards = (async (req, res, next) => {
+  try {
+    const cards = await Card.find({});
+    return res.status(200).send({ data: cards });
+  } catch (error) {
+    return next(error);
+  }
+});
+
+// POST /cards — создаёт карточку
+const postCard = (async (req, res, next) => {
+  try {
+    const { name, link } = req.body;
+    const card = await Card.create({ name, link, owner: req.user._id });
+    return res.status(201).send({ data: card });
+  } catch (error) {
+    return next(error);
+  }
+});
+
+// DELETE /cards/:cardId — удаляет карточку по идентификатору
+const deleteCard = (async (req, res, next) => {
+  try {
+    const { id } = req.params;
+    const card = await Card.findById(id);
+    if (!card) {
+      return res.status(404).json({ message: 'Карточка не найдена' });
+    }
```

Comment:
Можно лучше: передавать ошибку в обработчик ошибок.
По аналогии с отальными:
```suggestion
    if (!card) {
      throw new NotFoundError('Карточка не найдена');
    }
```
или
```suggestion
    if (!card) {
      return next(new NotFoundError('Карточка не найдена'));
    }
```